src/web/components/Byline.tsx

```diff
@@ -1,41 +1,78 @@
 import React from 'react';
 import { css, cx } from 'emotion';
 
-import { headline } from '@guardian/src-foundations/typography';
+import { headline, textSans } from '@guardian/src-foundations/typography';
 import { until } from '@guardian/src-foundations/mq';
+import { Special } from '@guardian/types';
 
 type Props = {
 	text: string;
 	palette: Palette;
+	format: Format;
 	size: SmallHeadlineSize;
 };
 
-const bylineStyles = (size: SmallHeadlineSize) => {
+const bylineStyles = (size: SmallHeadlineSize, format: Format) => {
+	const baseStyles = css`
+		display: block;
+		font-style: italic;
+	`;
+
 	switch (size) {
-		case 'large':
+		case 'large': {
+			if (format.theme === Special.Labs) {
+				return css`
+					${baseStyles};
+					${textSans.large()};
+					font-size: 24px;
```

Comment:
is `textSans.large()` not giving us font size already?

Reply:
It's not large enough 😞 

The issue is Labs is using `textSans` like it's a headline, but it wasn't designed for that

Reply:
😞